var/spack/repos/builtin/packages/rmgdft/package.py

```diff
@@ -16,6 +16,7 @@ class Rmgdft(CMakePackage, CudaPackage):
     maintainers("elbriggs")
     tags = ["ecp", "ecp-apps"]
     version("master", branch="master")
+    version("6.1.0", tag="v6.1.0", commit="471251b191abb5f6ffdca4333c1fcb2add3c52f2")
```

Comment:
It seems you duplicated the sha for v5.4.0

Reply:
Aargh. Thanks. Corrected.